ci/docker/runtime_functions.sh

```diff
@@ -983,6 +983,7 @@ sanity_check() {
 # $2 -> python_cmd: The python command to use to execute the tests, python or python3
 cd_unittest_ubuntu() {
     set -ex
+    source /opt/rh/rh-python35/enable
```

Comment:
what does this do?

Reply:
ok this seems to switch the python alias

Reply:
CentOS7 ships with Python 2 by default. 3.5 is opt in. This is the opt-in. You can find more info https://www.softwarecollections.org/en/

Reply:
Due to the way the CI runs stuff in the Dockerfile, we can't permanently opt-in when creating the Docker image. We can fix that later. For now need to opt in inside the runtime functions.